### PR TITLE
UX improvement during execution

### PR DESCRIPTION
**Pull request by blangel**
Provides hints as to navigation, avoids spurious character entry to start script display and ensures step count doesn't go negative.

tapsim-ux-improvement

### CODE REVIEW

**Comment by blangel** [comment]
@halseth rebased and ready for review

File: script/execute.go [hunk 1]
```diff
@@ -135,6 +107,11 @@ func Execute(pkScript []byte, witness [][]byte, interactive bool) error {
 		}
 
 		output.DrawTable(table, clearLines)
+		if currentStep > 1 {
```
Comment (halseth) [inline script/execute.go:110]: Maybe this should only be printed when `interactive == true`?

**Review by halseth — CHANGES_REQUESTED** [review]
Tested, works great! Just a minor comment.

Also needs another rebase.

File: script/execute.go (hunk 1 above)
Comment (blangel) [reply script/execute.go:110]: Definitely, good catch.

**Comment by blangel** [comment]
@halseth after rebase I see a need to refactor [this line](https://github.com/halseth/tapsim/pull/2/files#diff-97270cdfd55e0158093b029eec360d1bbf635079a81ed2bf5a767f0bb1b1bbf7L258).  This statement needs to either be appended to the table output in all cases after the initial print (otherwise it gets overwritten and not displayed; which could be the intent (?) but I don't think it is) or this should be extracted out of the table itself and printed a single time (like [these statements](https://github.com/halseth/tapsim/pull/2/files#diff-97270cdfd55e0158093b029eec360d1bbf635079a81ed2bf5a767f0bb1b1bbf7L75-L79), which seems more natural).  Let me know if you have a preferred approach.  I'll update this PR with what I think reasonable.

**Review by halseth — APPROVED** [review]
Works perfectly, LGTM 💯

**Pull request merged**